src/main/java/com/extendedclip/deluxemenus/config/DeluxeMenusConfig.java

```diff
@@ -1,1514 +1,1544 @@
+            Location loc = LocationUtils.deserializeLocation(c.getString(rPath + ".location"));
+            if (loc == null) {
+              DeluxeMenus.debug(
+                  DebugLevel.HIGHEST,
+                  Level.WARNING,
+                  "requirement at path: " + rPath + " has an invalid location. Valid Format is: <world>,<x>,<y>,<z>"
+              );
+            }
+            req = new IsNearRequirement(loc, c.getInt(rPath + ".distance"), invert);
+          } else {
+            DeluxeMenus.debug(
+                DebugLevel.HIGHEST,
+                Level.WARNING,
+                "Is Near requirement at path: " + rPath + " does not contain a location: or distance: entry"
+            );
+          }
+          break;
+        case HAS_META:
+        case DOES_NOT_HAVE_META:
+          if (!VersionHelper.IS_PDC_VERSION) {
+            DeluxeMenus.debug(
+                DebugLevel.HIGHEST,
+                Level.WARNING,
+                "Has Meta requirement is not available for your server version!"
+            );
+            break;
+          }
+          if (c.contains(rPath + ".key") && c.contains(rPath + ".meta_type") && c
+              .contains(rPath + ".value")) {
+            String metaKey = c.getString(rPath + ".key");
+            invert = type == RequirementType.DOES_NOT_HAVE_META;
+            req = new HasMetaRequirement(metaKey, c.getString(rPath + ".meta_type").toUpperCase(),
+                c.getString(rPath + ".value"), invert);
+          } else {
+            DeluxeMenus.debug(
+                DebugLevel.HIGHEST,
+                Level.WARNING,
+                "Has Meta requirement at path: " + rPath
+                    + " does not contain the key:, meta_type: and/or value: entries!"
+            );
+          }
+          break;
+        case STRING_LENGTH:
+          if (c.contains(rPath + ".input") && (c.contains(rPath + ".min") || c.contains(rPath + ".max"))) {
+            int min = c.getInt(rPath + ".min", 0);
+            Integer max = null;
+            if (c.contains(rPath + ".max")) {
+              max = c.getInt(rPath + ".max");
+            }
+            req = new StringLengthRequirement(c.getString(rPath + ".input"), min, max);
+          } else {
+            DeluxeMenus.debug(
+                DebugLevel.HIGHEST,
+                Level.WARNING,
+                "String length requirement at path: " + rPath + " does not contain an input: or one of (min: or max:)"
+            );
+          }
+          break;
+        case IS_OBJECT:
+          if (c.contains(rPath + ".input") && c.contains(rPath + ".object")) {
+            req = new IsObjectRequirement(c.getString(rPath + ".input"), c.getString(rPath + ".object"));
+          } else {
+            DeluxeMenus.debug(
+                DebugLevel.HIGHEST,
+                Level.WARNING,
+                "String length requirement at path: " + rPath + " does not contain an input: or one of (min: or max:)"
```

Comment:
`String length requirement at path: " + rPath + " does not contain an input or an object type`